package.json

```diff
@@ -38,14 +38,14 @@
     "classnames": "^2.2.6",
     "color": "^3.1.2",
     "design-system": "git://github.com/Wikia/design-system.git#22.14.0",
-    "i18next": "^19.3.2",
+    "i18next": "^19.7.0",
     "immutable": "^4.0.0-rc.12",
     "lodash": "^4.17.15",
     "mockdate": "^3.0.2",
     "prop-types": "^15.7.2",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
-    "react-i18next": "^11.3.3",
+    "react-i18next": "^11.7.1",
```

Comment:
I'm not sure if we should have this dependency both in `dependencies` and `peerDependencies`.

Reply:
Why not? All `dependencies`are excluded from build in rollup config as "externals", but I kept them here so docs & local development won't break.